composer.json

```diff
@@ -32,7 +32,7 @@
 		"phpcompatibility/phpcompatibility-wp": "2.1.0",
 		"sirbrillig/phpcs-changed": "2.2.7-beta-1@dev",
 		"sirbrillig/phpcs-variable-analysis": "2.7.0",
-		"wp-coding-standards/wpcs": "2.1.1"
+		"wp-coding-standards/wpcs": "^2.2"
```

Comment:
Don't we want to keep the dependency pinned so Renovate can pick up any changes?

Reply:
Renovate will pin it in a follow-up. If it doesn't beat me, I'd like to run our already-compliant files through again to make sure no new sniffs find anything. I'll pin it in that PR.